src/js/player.js

```diff
@@ -2199,6 +2199,91 @@ class Player extends Component {
     };
   }
 
+  /**
+   * Given a parent control, finds the first focusable child control
+   *
+   * @param {HTMLElement} parent - The control in which we'll search
+   * @return {HTMLElement | null} - Returns the first focusable control in the node list or null if not focusable control is found
+   */
+  getFirstFocusableControl_(parent) {
+    // Parent has no children, and the parent itself is not focusable, nothing to return
+    if (!parent || typeof parent.el !== 'function' || (!Dom.isFocusable(parent.el()) && !parent.children().length)) {
+      return null;
+    }
+
+    if (Dom.isFocusable(parent.el())) {
+      return parent.el();
+    }
+
+    const children = parent.children();
+    let focusableEl;
+
+    for (let i = 0; i < children.length; i++) {
+      const child = children[i];
+
+      focusableEl = this.getFirstFocusableControl_(child);
+
+      if (focusableEl) {
+        break;
+      }
+    }
+
+    return focusableEl;
+  }
+
+  /**
+   * Given a parent control, finds the last focusable child control
+   *
+   * @param {HTMLElement} parent - The control in which we'll search
+   * @return {HTMLElement | null} - Returns the last focusable control in the node list or null if not focusable control is found
+   */
+  getLastFocusableControl_(parent) {
```

Comment:
The reason for this method and the one above it is if someone added an element to the player DOM but their top level element isn't focusable but some child is, we should return that child element?